### PR TITLE
[DBAAS-817] Added cluster related information into metrics and code refactor

### PR DESCRIPTION
**Pull request by redhatHameed**
Screen Shot 2022-09-06 at 2 29 54 PM

Signed-off-by: Abdul Hameed <ahameed@redhat.com>

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA or in issue number have been completed
- [ ] Unit tests added that prove the fix is effective or the feature works 
- [ ] Documentation added for the feature
- [ ] all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

File: controllers/dbaasplatform_controller.go [hunk 1]
```diff
@@ -115,6 +118,16 @@ func (r *DBaaSPlatformReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	execution := PlatformInstallStart()
 	var platforms map[dbaasv1alpha1.PlatformsName]dbaasv1alpha1.PlatformConfig
 
+	consoleURl, err := util.GetOpenshiftConsoleUrl(ctx, r.Client)
```
Comment (jianrongzhang89) [inline controllers/dbaasplatform_controller.go:121]: Fix lint issue:  consoleURl -> consoleURL
GetOpenshiftConsoleUrl -> GetOpenshiftConsoleURL

File: controllers/util/openshift.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import (
+	"context"
+
+	v1 "github.com/openshift/api/config/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	VersionKeyName      = "version"
```
Comment (jianrongzhang89) [inline controllers/util/openshift.go:12]: Fix lint issue. Add comments to these vars or to the entire block.

File: controllers/util/openshift.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+	ClusterResourceName = "cluster"
+)
+
+// GetClusterIdVersion Returns the cluster id and cluster version by querying the ClusterVersion resource
+func GetClusterIdVersion(ctx context.Context, client k8sclient.Client) (string, string, error) {
```
Comment (jianrongzhang89) [inline controllers/util/openshift.go:17]: GetClusterIdVersion -> GetClusterIDVersion

File: controllers/dbaasplatform_controller.go (hunk 1 above)
Comment (redhatHameed) [reply controllers/dbaasplatform_controller.go]: done

File: controllers/util/openshift.go (hunk 2 above)
Comment (redhatHameed) [reply controllers/util/openshift.go:13]: done

File: controllers/util/openshift.go (hunk 3 above)
Comment (redhatHameed) [reply controllers/util/openshift.go]: done

**Review by jeremyary — APPROVED** [review]

**Pull request merged**